Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Description: Do not use warning flags that require a newer Clang (xenial has 6.0.0,
  see https://releases.llvm.org/6.0.0/tools/clang/docs/DiagnosticsReference.html)

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1406,7 +1406,7 @@ config("default_warnings") {
 
         # Ignore warnings about MSVC optimization pragmas.
         # TODO(thakis): Only for no_chromium_code? http://crbug.com/505314
-        "-Wno-ignored-pragma-optimize",
+        #"-Wno-ignored-pragma-optimize",
       ]
     } else if (use_xcode_clang) {
       cflags += [
